app/controllers/api/v2/smart_class_parameters_controller.rb

```diff
@@ -36,6 +36,7 @@ def show
         param :override, :bool, :desc => N_("Whether the smart class parameter value is managed by Foreman")
         param :description, String, :desc => N_("Description of smart class")
         param :default_value, String, :desc => N_("Value to use when there is no match")
+        param :hidden_value, :bool, :desc => N_("Value to display when the parameter should be hidden")
```

Comment:
Oh heck, I think I see what you mean.  `#hidden_value` is `*****` but `#hidden_value?` is the boolean attribute, right?

Remember that this here is the documentation for incoming parameters on create/update, *not* response parameters, so if a user specifies `hidden_value` then it would be there to update the boolean attribute.  I'll put another comment inline for the response/rabl part.